### PR TITLE
Updated onboarding card

### PR DESCRIPTION
**Pull request by adriannelee**
# Summary | Résumé

Related Zenhub card https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1285

# Test instructions | Instructions pour tester la modification
1. Are there any docs that you feel is _crucial_ to add here?
2. Are there docs included in this template that you think would be too much information for onboarding?

### CODE REVIEW

**Comment by adriannelee** [comment]
Melissa is updating the Policy 101 doc and should be finished at the end of this week.

**Review by yaelberger-commits — APPROVED** [review]
Only not sure about the heading "Learn about stakeholders" for the platform strategy. Otherwise all changes look great!

**Review by amazingphilippe — APPROVED** [review]
Looks good, and makes me think we could have better documentation around design!

**Review by jzbahrai — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md [hunk 1]
```diff
@@ -7,51 +7,57 @@ assignees: ''
 
 ---
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:9]: Do we want to provide a time frame for this to be completed? We usually told our new onboarded employees (on the dev side at least) to take a week to digest information and do this card: do not even do dev contributions yet, just take the week.

File: .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md [hunk 2]
```diff
@@ -7,51 +7,57 @@ assignees: ''
...
+- [ ] [Team rituals](https://docs.google.com/document/d/1h4kTrfP257J4QVeSPNAr2-tgV-SM2BbyBbNTUb3WPdc/edit)
+- [ ] [Types of documentation](https://docs.google.com/spreadsheets/d/1sondKNE4bBRN8D6CEYx138vpHVojfOsO71mAz6qGwqs/edit#gid=0)
 
-#  Read and discover **(the following are suggested in order of priority. Reading all of it is not mandatory)**
+## Learn about stakeholders
+- [ ] [Platform business unit - Strategy 2022-23](https://docs.google.com/presentation/u/0/d/1TWAi89DsG696pT4bmZV54Mjn_KP0TzP9R6_elkUDbqU/edit)
 
-- [ ] [Overview of Notify product](https://docs.google.com/document/d/1naerXecHaMR6-udeYiny5a_84CvIKwITAlwR8ZwsIZY/edit#) 
-- [ ] [Notify Scavenger Hunt](https://docs.google.com/document/d/1MmKBZqMNpsLU0TlURGCyOlByZHGfZccteuYdGZ9amwM/edit#heading=h.sxqbcn4k0lmo)🕵️‍♀️
-- [ ] [Directory of links](https://docs.google.com/document/d/15dk3fz6BwX0_ZwHyCXMDYIWCszXSoFarqydKQNeeyx0/edit)
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:22]: As we are removing this document, there are links that it contained worth sharing.

For Devs:

* [Technical Playbook](https://cds-snc.github.io/technical-playbook-manuel-technique/)
* [Code review guidelines](https://github.com/cds-snc/cra-claim-tax-benefits/blob/main/docs/CODE-REVIEW-GUIDELINES.md)

For all:

* [Service-Level Agreement for GC Notify](https://docs.google.com/document/d/1JAuWT6XwYUFRetSFffcfCJOuUryW1D8yPeAa4NkWWgs/edit#heading=h.pbh32eniyzdc)

Comment (adriannelee) [reply .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:22]: Nice! I added the two devs docs to this onboarding doc (linked in the Deeper Dive section) and added the SLA as a deeper dive item for anyone.

Comment (adriannelee) [reply .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:22]: It took me a second to figure out how to do it in an existing PR :D .... You should see it now! .. here?

File: .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md [hunk 3]
```diff
@@ -64,4 +64,5 @@ _Delete section if not a dev_
 - [ ] Platform admin @jimleroyer or @jzbahrai
 
 # Done when
-New team member says so! Should have had the time to digest info, connect with team and the access needed are granted, **and [audit document](https://docs.google.com/spreadsheets/d/1nqRmotJvNY_ftNwr5R7_tk2HKrveNRBA-R77kNTU-FM/edit#gid=0) has been updated**.
+- [ ] New team member says so! Take the week to digest these items. Don't feel pressured to contribute during your onboarding.
+- [ ] [Audit document](https://docs.google.com/spreadsheets/d/1nqRmotJvNY_ftNwr5R7_tk2HKrveNRBA-R77kNTU-FM/edit#gid=0) has been updated
```
Comment (jimleroyer) [inline .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:68]: ok I am impressed you added this one!

File: .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md (hunk 2 above)
Comment (jimleroyer) [reply .github/ISSUE_TEMPLATE/-onboarding--welcome-a-new-team-member.md:22]: I see the SLA change but not the technical playbook and code review guidelines! 😵

**Review by jimleroyer — APPROVED** [review]

**Comment by jimleroyer** [comment]
Thank you for the nice updates!

**Pull request merged**